app/models/application_record.rb

```diff
@@ -18,7 +18,7 @@ def self.estimated_count
     query = sanitize_sql_array([QUERY_ESTIMATED_COUNT, table_name, table_name])
     result = connection.execute(query)
 
-    count = result.first["count"]
+    count = result.max_by(&:values)["count"] # sometimes is an array of hashes, for ex. [{"count" => 0}, [{"count" => 1000}]
```

Comment:
@rhymes any thoughts on why this might occur?

Reply:
When I was comparing development vs production, the dev Postgres `result` had 1 tuple, whereas the production had 2 tuples. Not sure if that helps?

Reply:
Woah! This is super weird. What happens if you run the query manually through production's `heroku pg:psql` ? Do you have two values?

Reply:
Hmm yeah! Two rows:
```
 count
--------
      0
 407020
(2 rows)
```